specification/maps/data-plane/AsyncBatchManagement/preview/2024-04-01-preview/asyncBatchManagement.json

```diff
@@ -66,8 +66,8 @@
       "required": true,
       "x-ms-parameter-location": "client"
     },
-    "OperationId": {
-      "name": "operationId",
+    "Id": {
+      "name": "id",
```

Comment:
nit, the name for path parameter does not affect REST API

Reply:
This change is not necessary for compliance with Azure REST API Guidelines. You could leave the parameter name as `operationId` and that would be fine. It's also fine to change it to `id`, so up to you whether you keep this change or revert it.